src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,49 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    this.hidden = false;
```

Comment:
Animal class doesn't have this property, only `Herbivore` has
![image](https://github.com/user-attachments/assets/e9f32789-391c-4a65-a749-8ad9ecfe8e61)